cmd/signing.go

```diff
@@ -31,31 +32,29 @@ func init() {
 	SigningCommand = cli.Command{
 		Name:  "signing",
 		Usage: "Provides several tools useful for out-of-band signing",
+		Before: func(c *cli.Context) error {
+			_ = log.SetLogLevel("*", "fatal") // disable the regular logger
```

Comment:
Thoughts on going through our existing `keep-common` logging abstraction here and doing a `logging.Configure("*", "fatal")`? My thinking is it'll be easier to bridge this if and when we bump our logging dependencies to a different major version or so (e.g. with keep-network/keep-common#30).

Reply:
Addressed in the follow-up PR:  #568